### PR TITLE
adding mysql database facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding missing MySQL database facts module

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
azure_rm_mysqldatabase_facts

##### ANSIBLE VERSION
2.6

### CODE REVIEW

**Comment by Fred-sun** [comment]
@zikalino does this PR ready for review, please help confirm? Thanks!

**Comment by Fred-sun** [comment]
@zikalino Could you help to confirm the PR ready for review or not? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase_facts.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_mysqldatabase_facts
+version_added: "2.7"
+short_description: Get MySQL Database facts.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase_facts.py]: Azure

File: lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase_facts.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                required=True
+            ),
+            name=dict(
+                type='str'
+            )
+        )
+        # store the results of the module operation
+        self.results = dict(
+            changed=False,
+            ansible_facts=dict()
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase_facts.py]: remove ansible_facts

**Pull request merged**